### PR TITLE
delete icon

### PR DESCRIPTION
**Pull request by Mackiovello**
As described in #21, the icon makes it impossible to blend to the "user" token.

### CODE REVIEW

**Comment by warpech** [comment]
I agree this is a sound decision, but please let's add a launchpad icon instead?

**Comment by Mackiovello** [comment]
> I agree this is a sound decision, but please let's add a launchpad icon instead?

Sure, I'll fix a launchpad icon

**Comment by Mackiovello** [comment]
Wait a second... Why would you want to have a launchpad icon for the editor? What would happen when you click on it? There's no real page to go to. It wouldn't make sense to open the editor since it's only able to edit the page it's on.

**Comment by warpech** [comment]
Hmm... You are right, it cannot just open the editor. Definitely we can put there something that's helpful, for example a page saying:

> You are running **CompositionEditor**. Navigate to any page, press CTRL + E and compose your views.

> Here are some tips for the app authors: [HTML Views Blending Guidelines](https://docs.starcounter.io/guides/web-apps/html-view-guidelines/)

**Comment by Mackiovello** [comment]
> Hmm... You are right, it cannot just open the editor. Definitely we can put there something that's helpful, for example a page saying:

This is now implemented with #28, is there anything preventing me from merging and releasing a new version of CompositionEditor?

**Comment by warpech** [comment]
This change looks good to me.

Please have a look at https://github.com/StarcounterApps/CompositionEditor/issues/29. 

@tomalec as the owner of Blending TF should give it a final review before releasing a version.

File: src/CompositionEditor/wwwroot/CompositionEditor/Master.html [hunk 1]
```diff
@@ -3,77 +3,6 @@
 
 <template>
     <template is="dom-bind">
-        <button slot="compositioneditor/icon" active$="{{model.editingMode$}}" on-click="toggleEditing" aria-label="Brush icon">
-            <svg modified$="{{model.layoutModified$}}" viewBox="0 0 74 76" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-                <use xlink:href="/CompositionEditor/images/layout-editor.svg#Page-1"/>
-            </svg>
-        </button>
-
         <compositioneditor-inserter slot="compositioneditor/inserter" active="{{model.editingMode$}}"></compositioneditor-inserter>
```
Comment (tomalec) [inline src/CompositionEditor/wwwroot/CompositionEditor/Master.html]: Do we still need this ` active="{{model.editingMode$}}"` and Polymer here, as we no longer have button to bind to?

Comment (Mackiovello) [inline src/CompositionEditor/wwwroot/CompositionEditor/Master.html]: Fixed in 	50b1974

**Comment by Mackiovello** [comment]
Is there more that needs fixing @tomalec?

**Comment by Mackiovello** [comment]
Merged to develop. Closing

**Pull request closed without merging**